src/components/update-profile/UpdateProfile.jsx

```diff
@@ -1,287 +1,89 @@
-import SideBarProfile from "./sidebarProfile";
 import "./profile.css";
-import { useState, useEffect } from "react";
-import avatar from "./Ellipse 2.png";
+import { useEffect, useState } from "react";
 import Header from "../admin-header/Header";
+import UpdateProfileForm from "./UpdateForm";
 import { ToastContainer, toast } from "react-toastify";
 import { useNavigate } from "react-router-dom";
+import Sidebar from "../admin-sidebar/SideBar";
+import { useJwt } from "react-jwt";
 
-const UpdateProfile = () => {
-  let localUser;
-  if (localStorage.getItem("user")) {
-    localUser = JSON.parse(localStorage.getItem("user"));
-  }
-  const [user, setUser] = useState(
-    localUser
-      ? localUser
-      : {
-          firstName: "",
-          lastName: "",
-          nationId: "",
-          phone: "",
-          role: "",
-          gender: "",
-          category: "",
-          bio: "",
-          country: "",
-          city: "",
-          address: "",
-        }
-  );
-
-  //variables for input fields
-  const [firstName, setFirstName] = useState(user ? user.firstName : "");
-  const [lastName, setLastName] = useState(user ? user.lastName : "");
-  const [nationId, setNationId] = useState(user ? user.nationId : "");
-  const [phone, setPhone] = useState(user ? user.phone : "");
-  const [role, setRole] = useState(user ? user.role : "");
-  const [gender, setGender] = useState(user ? user.gender : "");
-  const [category, setCategory] = useState(user ? user.category : "");
-  const [bio, setBio] = useState(user ? user.bio : "");
-  const [country, setCountry] = useState(user ? user.country : "");
-  const [city, setCity] = useState(user ? user.city : "");
-  const [address, setAddress] = useState(user ? user.address : "");
+const token = localStorage.getItem("token");
 
-  const handleSubmit = (e) => {
-    e.preventDefault();
-    if (validate()) {
-      setUser({
-        firstName,
-        lastName,
-        nationId,
-        phone,
-        role,
-        gender,
-        category,
-        bio,
-        country,
-        city,
-        address,
-      });
-      localStorage.setItem("user", JSON.stringify(user));
-      toast.info("Profile Updated", {
-        theme: "colored",
-      });
-    }
-  };
-  const HandleReset = () => {
-    setAddress("");
-    setBio("");
-    setCategory("");
-    setCity("");
-    setCountry("");
-    setFirstName("");
-    setGender("");
-    setLastName("");
-    setNationId("");
-    setPhone("");
-    setRole("");
-  };
+const UPDATE_PROFILE_URL = process.env.BACKEND_URL;
```

Comment:
It is failing to get data from DB, because id of the user logged in don't match with any record in Profile table. the PR to fix issues on the BE not yet merged.

Reply:
Ok, let's first get it merged, I'll revert to you shortly after